src/pk/ecc/ecc_ssh_ecdsa_encode_name.c

```diff
@@ -0,0 +1,57 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis
+ *
+ * LibTomCrypt is a library that provides various cryptographic
+ * algorithms in a highly modular and flexible manner.
+ *
+ * The library is free for all purposes without any express
+ * guarantee it works.
+ */
+#include "tomcrypt_private.h"
+
+/**
+   @file ecc_ssh_ecdsa_encode_name.c
+   Curve/OID to SSH+ECDSA name string mapping per RFC5656
+   Russ Williams
+*/
+
+int ecc_ssh_ecdsa_encode_name(char *buffer, unsigned long buflen, const ecc_key *key)
```

Comment:
To keep the style used on other places in libtomcrypt it should be `unsigned long *buflen` (but as it is a private function maybe we can leave it as it is, @sjaeckel ?)

Reply:
Done, and it writes out the length into *buflen.